src/components/newcases.js

```diff
@@ -0,0 +1,140 @@
+import React, {useState, useEffect} from 'react';
+
+function RateOfNewCases(props) {
+  const [casesMap, setCasesMap] = useState(0);
+  const [length, setLength] = useState(0);
```

Comment:
```
const length = props.timeseries.length
```

Reply:
This needs to be in state as `length` is changing as per the state.

Reply:
But it's always equal to props.timeseries.length. Is there a case where it differs? If not, why store it in a state hook?

Reply:
Removed this.